gossip/gossip.go

```diff
@@ -90,7 +90,7 @@ const (
 	defaultNodeCount = 1000
 
 	// ttlNodeIDGossip is time-to-live for node ID -> address.
-	ttlNodeIDGossip = 0 * time.Second
+	ttlNodeIDGossip = 0
```

Comment:
Constants *may* be untyped, but where a named type like `time.Duration` applies they probably should be typed (either with `const ttlNodeIDGossip time.Duration = 0` or `* time.Second` as before).